README.md

```diff
@@ -126,8 +126,11 @@ The `Connection.connect()` function can take the following additional options:
   convenient for human inspection while still being usable by
   geospatial data manipulation libraries.
 
-- `region`: Currently, the only supported Wherobots compute region is `aws-us-west-2`,
-  in AWS's Oregon (`us-west-2`) region.
+- `region`: You must also specify in which region your SQL session should execute
```

Comment:
See https://github.com/wherobots/wherobots-python-dbapi/pull/38/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R72-R75, we might want a similar note here.

Reply:
done!